draft-yasskin-http-origin-signed-responses.md

```diff
@@ -386,6 +386,58 @@ extended diagnostic notation from {{?I-D.ietf-cbor-cddl}} appendix G:
 ]
 ~~~
 
+## Loading a certificate chain ## {#cert-chain}
+
+The resource at a signature's `certUrl` MUST have the
+`application/cert-chain+cbor` content type, MUST be canonically-encoded CBOR
+({{canonical-cbor}}), and MUST match the following CDDL:
+
+~~~cddl
+cert-chain = [
+  "📜⛓", ; U+1F4DC U+26D3
+  + {
+    cert: bytes,
+    ? ocsp: bytes,
```

Comment:
Oh, also, this is a simplification of the `status_request` TLS extension that omits the layer containing a `CertificateStatusType`. Is that the right choice?

Reply:
I believe so, at least as we discussed. This might be an area of future change.

The practical reality is that OCSP multi-stapling is rather inefficient for the browser case, given that "Every Modern Browser" (tm) effectively has an out-of-bands mechanism for dealing with intermediate revocation, treating it akin to root revocation, as well as heavily caching intermediate results (e.g. in the case of macOS, via an out-of-band datafile containing the revocation status of all intermediates, in the case of Windows, through background-fetching CRLs that cover CAs)

From a PKI pursist standpoint, however, you could try and argue that such systems don't handle nested sets of name-constrained subordinate CAs, and for those cases (and, effectively, those cases alone), it makes sense to allow OCSP responses for intermediates to be stapled.

Reply:
I'm going to omit the "nested sets of name-constrained subordinate CAs" case from this draft, and if someone tells me to add it, I'm going to make them write the wording that distinguishes certificates that need OCSPs from ones that don't.